### DISCUSSION TITLE
Please add Sony Sab

### DISCUSSION THREAD

**Discussion by alizainsoomro321** [iptv-org/community#1090] [Q&A]
Hi there,

Could you please add the Sony Sab channel?

Thank you

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).

**Reply by alizainsoomro321** [reply]
Sure, please prioritize. I’ve been waiting almost 9 months.

**Reply by alizainsoomro321** [reply]
I have created another issue as you said, [https://github.com/iptv-org/iptv/issues/20283](https://github.com/iptv-org/iptv/issues/20283)

Please try to resolve it as soon as possible.

Thank you
